notebook/README.md

```diff
@@ -0,0 +1,37 @@
+# notebook
+
+A simple example using a Jupyter notbook with Cog
+
+## Usage
+
+First, make sure you've got the [latest version of Cog](https://github.com/replicate/cog#install) installed.
+
+Build the image:
+
+```sh
+cog build
+```
+
+Run the Jupyter notebook server with Cog:
```

Comment:
```suggestion
Run the Jupyter Notebook server with Cog:
```

Capitalisation of a lone `notebook` is a little inconsistent on the project's site, but it's always capitalised when written in full.